crates/cext/src/transpiler/passes/vf2.rs

```diff
@@ -118,40 +120,180 @@ pub unsafe extern "C" fn qk_vf2_layout_result_free(layout: *mut VF2LayoutResult)
     }
 }
 
+/// A set of configurations for the VF2 layout passes.
+///
+/// See the setter methods associated with this `struct` for the available configurations.
+pub struct VF2LayoutConfiguration(Vf2PassConfiguration);
+/// @ingroup QkVF2LayoutConfiguration
+/// Create a new configuration for the VF2 passes that runs everything completely unbounded.
+///
+/// Call ``qk_vf2_layout_configuration_free`` with the return value to free the memory when done.
+#[no_mangle]
+#[cfg(feature = "cbinding")]
+pub extern "C" fn qk_vf2_layout_configuration_new() -> *mut VF2LayoutConfiguration {
+    Box::into_raw(Box::new(VF2LayoutConfiguration(
+        Vf2PassConfiguration::default_unbounded(),
+    )))
+}
+/// @ingroup QkVF2LayoutConfiguration
+/// Create a new configuration for the VF2 passes that runs everything completely unbounded.
+///
+/// # Safety
+///
+/// Behavior is undefined if ``config`` is a non-null pointer, but does not point to a valid,
+/// aligned ``VF2LayoutConfiguration`` object.
+#[no_mangle]
+#[cfg(feature = "cbinding")]
+pub unsafe extern "C" fn qk_vf2_layout_configuration_free(config: *mut VF2LayoutConfiguration) {
+    if !config.is_null() {
+        if !config.is_aligned() {
+            panic!("Attempted to free a non-aligned pointer.");
+        }
+        let _ = unsafe { Box::from_raw(config) };
+    }
+}
+/// @ingroup QkVF2LayoutConfiguration
+/// Limit the numbers of times that the VF2 algorithm will attempt to extend its mapping.
+///
+/// @param config The configuration to update.
+/// @param limit The number of attempts to allow.  Set to a negative number to have no bound.
+///
+/// # Safety
+///
+/// Behavior is undefined if `config` is not a valid, aligned, non-null pointer to a
+/// `VF2LayoutConfiguration`.
+#[no_mangle]
+#[cfg(feature = "cbinding")]
+pub unsafe extern "C" fn qk_vf2_layout_configuration_call_limit(
```

Comment:
I'm wondering if `qk_vf2_layout_configuration_set_<...>` would be a more better naming scheme, to be explicit about setting? Initially I thought this would return the `call_limit` 🤔

Reply:
I changed these over in 882f112dc